src/configuration/configuration_controller.py

```diff
@@ -80,9 +76,27 @@ def deploy_blockchain(self, blockchain_name):
         """
 
         try:
-            cmd = [self._multichain_d,blockchain_name, self._multichain_daemon]
-            subprocess.run(cmd, timeout=5)
-        except ProcessExecutionError as err:
-            print(err.args[3])
+            cmd = self._multichain_d_arg+[blockchain_name, self._multichain_daemon]+[self._data_dir_arg+self._params_path]
+            run(cmd, timeout=5, cwd=self._install_path)
+        except CalledProcessError as err:
+            print(err.stderr)
         except Exception as err:
             print(err)
+
+    def validate_params_path(self,path):
+        """
+        Validates the the params path provided by the user
+        :return: a valid path of params.dat location
+        """
+        if os.path.exists(path):
+            return path
+        return str(Path.home())+self._multichain_path
```

Comment:
when combining paths, it is best to use os.path.join() to maintain cross compatibility